src/components/RelevantRightNow.js

```diff
@@ -50,7 +49,10 @@ function isTimeBetween(start, end) {
   // used different date to capture screenshots
   // using now means all will be filtered out
   const currentTime = new Date();
-  return isWithinInterval(currentTime, {
+  // return false if either the start or end is undefined
+  const range_defined = start !== undefined && end !== undefined;
```

Comment:
might be a design thing but im wondering if the cards now look awkward cause they're different sizes

Reply:
That's a good point. I think ideally they'll all have buttons and descriptions, which will make them at least closer in size. 

Might be nice to have them all the same height too though if we have time.